src/components/CustomModal.js

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
 import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
 
+const renderParagraphs = function(paragraphs) {
+    if (paragraphs) {
+            return paragraphs.map((paragraph, index) => (
+                <p key={index} >{paragraph}</p>
+            ));
+        }
+    },
+
+    renderUrlButtons = function(links) {
+        if (links) {
+            return links.map((link, index) => (
+                <a key={index} target="_blank" href={link.url}>
+                    <Button className="pointer"> {link.label} </Button>
+                </a>
+            ));
+        }
+    };
```

Comment:
Those two functions were created in order to modulate more this component, following the DRY principle. We map the paragraphs and links arrays, in order to create the tags needed. The tag "template" is defined on the function, so if we need to change it someday, we only need to change it there.